### ISSUE TITLE
Character Movement

### ISSUE THREAD

**Issue by 9mina** [MINAqwq/VR-NetRack#1]
# TODO
- [ ] VR
- - [ ] Custom height
- - [x] Movement
- [x] Keyboard
- - [x] Movement

Both users can walk through the world (keyboard users with WASD) and VR user should be able to change the height of their "head".
